### ISSUE TITLE
Help icons in key areas

### ISSUE THREAD

**Issue by mikeallbutt** [DiscipleTools/disciple-tools-mobile-app#91]
Like [this mockup](https://xd.adobe.com/view/7080ce9f-27ae-4018-449e-33b6bcc61293-06b1/screen/673eeddc-b296-4467-bcf7-0201a143ac9c/Group-Details-11), have a help icon that displays the help modal.

<img width="258" alt="Screenshot 2019-11-07 04 56 04" src="https://user-images.githubusercontent.com/43966676/68383322-0eaa2d80-011b-11ea-81bc-47e04f8f4960.png">

**Comment by mikeallbutt** [comment]
I would be happy to try to implement this and others but will need some pointers on how to do it - or better still, if someone can give me an example of one (this one above) and I can then go through the app and put others in at key locations (one per screen might be good).

**Comment by raschdiaz** [comment]
Hi @mikeallbutt @zdmc23 , we have some questions about this:

- The help icons shows only in 'detail mode'?
- The 'read the documentation' button have to appear in this dialogs?.
- The showed texts have to change according to app language?.

**Comment by mikeallbutt** [comment]
Yes the help text would need to change according to the app language interface.

**Comment by mikeallbutt** [comment]
One place that really needs explanation is on the login screen. I get asked what to put into the URL field. Maybe having a (?) icon on the right wide of the URL field that when clicked pops up a message saying what to put here i.e the url of your DT instance.
I know that this info for a URL field might be obvious for some - but now all find it as obvious and need more guidance.

**Comment by zdmc23** [comment]
@mikeallbutt could you please document your discussion about context menu help from our D.T Mobile App call?

**Comment by ericchen59** [comment]
[DTMA-11]

**Comment by mikeallbutt** [comment]
I think this is still a good idea for a few places that most users will need help understanding - like the church health diagram.
